templatefuncs.go

```diff
@@ -29,13 +32,17 @@ var fileModeTypeNames = map[fs.FileMode]string{
 // functions.
 func NewFuncMap() template.FuncMap {
 	return template.FuncMap{
+		"compact":          compactTemplateFunc,
+		"concat":           slices.Concat[[]any],
```

Comment:
This needs to be more generic so that it can accept `[]string` and other slices. Note that Go cannot automatically convert `[]any` to `[]string`. See https://blog.merovius.de/posts/2018-06-03-why-doesnt-go-have-variance-in/ if you want the gory details.